libraries/AP_HAL/board/linux.h

```diff
@@ -211,6 +211,21 @@
 #define HAL_INS_DEFAULT HAL_INS_HIL
 #define HAL_BARO_DEFAULT HAL_BARO_HIL
 #define HAL_COMPASS_DEFAULT HAL_COMPASS_HIL
+#elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_OCPOC_ZYNQ
+#define HAL_BOARD_LOG_DIRECTORY "/var/APM/logs"
+#define HAL_BOARD_TERRAIN_DIRECTORY "/var/APM/terrain"
+#define HAL_INS_DEFAULT HAL_INS_MPU9250_SPI
+#define HAL_INS_DEFAULT_ROTATION ROTATION_NONE
+#define HAL_INS_MPU9250_NAME "mpu9250"
+#define HAL_BARO_DEFAULT HAL_BARO_MS5611_SPI
+#define HAL_BARO_MS5611_NAME "ms5611"
+#define HAL_COMPASS_DEFAULT HAL_COMPASS_OCPOC_ZYNQ
+#define OCPOC_DEFAULT_RCIN_SBUS 1
+/* HMC5843 can be an external compass */
+#define HAL_COMPASS_HMC5843_I2C_BUS 4
+#define HAL_COMPASS_HMC5843_I2C_ADDR 0x1E
+#define AP_ARMING_COMPASS_OFFSETS_MAX 1000
+#define COMPASS_OFFSETS_MAX 1000
```

Comment:
These two macros don't exist anymore, it is now a parameter, defaults to 850, but you can override it with AP_COMPASS_OFFSETS_MAX_DEFAULT.